### ISSUE TITLE
the image paths are changed

### ISSUE THREAD

**Issue by charliedua** [realpython/python-guide#913]
The folder which contains the images is moved resulting in the images not being shown, else a broken image is shown.

**Comment by dbader** [comment]
Thanks @charliedua—which pages on https://docs.python-guide.org/ are affected?

**Comment by charliedua** [comment]
It is actually on GitHub rather than the website itself, the preview of the *.rst files contains wrong path to images.

**Comment by dbader** [comment]
Ah I see what you mean...yeah not sure if this is something we absolutely need to fix (as long as the website is ok) but I'd be open to a PR that resolves this.

**Comment by charliedua** [comment]
I can try to write a small python script which adds "/docs" before the "/_static" to all *.rst files eventually fixing the path. Yeah I can do that.

**Comment by dbader** [comment]
@charliedua That will break the live site. Fixing the GitHub preview for images seems more trouble than it's worth at this point IMO.

**Comment by charliedua** [comment]
Yeah that I think it is not only affecting the images, its links as well, anyway I
 wrote a simple python script which fixes the image links and will upgrade it to fix the links as well, however if it is breaking the original site we can make a branch for people who read it on GitHub?

**Comment by dbader** [comment]
> However if it is breaking the original site we can make a branch for people who read it on GitHub?

Mmh interesting idea but I don't have the bandwidth to maintain a parallel branch. Any solution you come up with would have to keep everything in one repo on one branch so we can use the same codebase on the website.
